src/components/container.tsx

```diff
@@ -0,0 +1,12 @@
+import {colors} from './colors';
+import styled from 'styled-components/native';
+import {Dimensions} from 'react-native';
+
+export const Container = styled.View`
+  display: flex;
+  align-items: center;
+  background-color: ${colors.white};
+`;
+
+export const ScreenWidth = Dimensions.get('screen').width;
```

Comment:
this shoud be in the theme, like the colors: 

utils/theme.ts

const theme = {
   colors: ....
   screenWidth....
}

export default theme